VladimirPopov_WebForms/templates/webforms/fields/select_checkbox.phtml

```diff
@@ -0,0 +1,121 @@
+                    checkbox
+                '
+                style='<?= $this->getFieldStyle() ?>'
+            >
+                <input type='checkbox'
+                    name='<?= $this->getFieldName() ?>[]'
+                    id='<?= $this->getFieldId() . $i ?>'
+                    value="<?= htmlspecialchars($option["value"]) ?>"
+                    class='
+                        checkbox__field
+                        <?= $validate ?>
+                        <?= $validate_min ?>
+                        <?= $validate_max ?>
+                    '
+                    <?= $checked ?>
+                    aria-labelledby="
+                        <?= $this->getFieldId() . $i ?>
+                        _label<?php if(!$this->getField()->getHideLabel()){?>
+                        label_<?= $this->getFieldId() ?><?php } ?>
+                    "
+                    <?= htmlspecialchars($this->getField()->getCustomAttributes())?>
+                />
+                <svg
+                    class="checkbox__icon checkbox__icon--checked"
+                    role="presentation"
+                    focusable="false"
+                >
+                    <use href="<?= $block->escapeUrl($block->getViewFileUrl('images/icons-sprite.svg#checked')) ?>"/>
+                </svg>
+                    <svg
+                    class="checkbox__icon checkbox__icon--unchecked"
+                    role="presentation"
+                    focusable="false"
+                >
+                    <use href="<?= $block->escapeUrl($block->getViewFileUrl('images/icons-sprite.svg#unchecked')) ?>"/>
+                </svg>
+                <label
+                    id="<?= $this->getFieldId() . $i ?>_label"
+                    for='<?= $this->getFieldId() . $i ?>'
+                    class="checkbox__label"
+                >
+                    <?= $label ?>
+                </label>
+                <?php
+                    $tooltip = $this->getField()->getTooltip($option["value"]);
+                    if ($tooltip) {
+                        echo $this->getLayout()->createBlock(
+                            'VladimirPopov\WebForms\Block\Script\Tooltip',
+                            null
+                        )->setField($this->getField())->setTooltip($tooltip)->toHtml();
+                    }
+                ?>
+            </div>
+            <?php
+        }
+    }
+    ?>
+</div>
+<?php if($validate_min) {
+    $validate_message = __('Please check at least %1 options', $this->getField()->getValue('options_checkbox_min'));
+    if (!empty($this->getField()->getValue('options_checkbox_min_error_text')))
+        $validate_message = $this->getField()->getValue('options_checkbox_min_error_text');
+    ?>
+    <div class="validation-advice"
+         id="advice-validate-field-options-min-<?= $this->getField()->getId()?>-<?= $this->getFieldId(). $i?>"
+         style="display:none"><?= $validate_message ?></div>
```

Comment:
you can use `display-none` class instead `style="display:none"`

Check atomic classes in `vendor/snowdog/theme-frontend-alpaca/Snowdog_Components/components/Atoms/_atomic/_atomic.scss`

Reply:
fix other code below too